### PR TITLE
NTO-18 Create Product Endpoint

### PR DESCRIPTION
**Pull request by rhedwan**

### CODE REVIEW

File: product/crud.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+def create_plans(database_session: Session, request: PlansBase):
+
+    new_plan = Plans(
+        name = request.name,
+        description = request.description,
+        duration = request.duration,
+        price  = request.price
+    )
+    database_session.add(new_plan)
+    database_session.commit()
```
Comment (madvirus-ops) [inline product/crud.py]: I think we should all us db for a uniform work

File: product/router.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from fastapi import APIRouter, Depends
+from sqlalchemy.orm.session import Session
+from .schemas import PlansBase, PlansListView
+from .crud import create_plans, get_all_plans, get_a_plan, delete_a_plan
+from config.database import get_db
+
+
+router = APIRouter(
+    prefix="/plans",
```
Comment (madvirus-ops) [inline product/router.py]: Specify a version

File: product/schemas.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    description :str
+    duration : int
+    price : int
+    create_at : str
+
+
+
+class PlansListView(BaseModel):
+    id : UUID
+    pkid : int
```
Comment (madvirus-ops) [inline product/schemas.py]: Remove the pk from the response model

**Review by madvirus-ops — CHANGES_REQUESTED** [review]
Make changes as specified

File: product/schemas.py (hunk 3 above)
Comment (rhedwan) [reply product/schemas.py]: fixed already

File: product/crud.py (hunk 1 above)
Comment (rhedwan) [reply product/crud.py]: okay

**Pull request merged**